docs/MakeOTFUserGuide.md

```diff
@@ -239,30 +233,25 @@ Versions of MakeOTF prior to FDK 2.5 used a similar syntax in the FontMenuNameDB
 |`c=<Windows compatible menu name>`|
 |`c=1,<Macintosh compatible menu name>`|
 
-If the key `c=` is used, then MakeOTF will build the older style name table. If the keys `l=` or `m=` are present, it will build the newer style name table . If none of these are present, then there is no difference in how the name table is built.
+If the key `c=` is used, then `makeotf` will build the older style name table. If the keys `l=` or `m=` are present, it will build the newer style name table . If none of these are present, then there is no difference in how the name table is built.
 
 ## **GlyphOrderAndAliasDB** (GOADB)
 
-The GOADB file is used to rename and to establish an order for the glyphs in a font.
-It is a simple text file with one line per glyph name. Each line contains at least two fields,
-and optionally a third field.
-Fields within a line are tab separated
-(technically any amount of a whitespace works but a single ASCII TAB is preferred).
-Blank lines are ignored.
-Lines beginning with `#` are comments and are also ignored.
-The first field is the final glyph name to be used in the output font.
-The second field is the ‘friendly’ name used in the source font data. 
-The third field is a Unicode value, specified in the form `uniXXXX` or `uXXXX[XX]` (see [note](#unicode_note)). 
-One may specify more than one Unicode value for a glyph by giving a comma separated list of values, for example: `uni0020,uni00A0`. 
+The GOADB file is used to rename and to establish an order for the glyphs in a font. It is a simple text file with one line per glyph name. Each line contains at least two columns, and optionally a third column.  
+Colums are tab-separated (technically any amount of a whitespace works but a single tab character is preferred).  
```

Comment:
And "recommended" rather than "preferred".

Reply:
I just checked this – indeed, any whitespace separation works. Not sure why we’d recommend one over the other then?

Reply:
I'm fine with getting rid of that too. "Preferred" didn't make sense -- what would be doing the preferring? If we want to recommend, we can recommend. Or we can delete that.